localstack/services/sns/provider.py

```diff
@@ -308,25 +300,6 @@ def list_tags_for_resource(
         tags = store.sns_tags.setdefault(resource_arn, [])
         return ListTagsForResourceResponse(Tags=tags)
 
-    def delete_platform_application(
-        self, context: RequestContext, platform_application_arn: String
-    ) -> None:
-        call_moto(context)
-
-    def delete_endpoint(self, context: RequestContext, endpoint_arn: String) -> None:
-        call_moto(context)
-
-    def create_platform_application(
-        self, context: RequestContext, name: String, platform: String, attributes: MapStringToString
-    ) -> CreatePlatformApplicationResponse:
-        # TODO: validate platform
-        # see https://docs.aws.amazon.com/cli/latest/reference/sns/create-platform-application.html
-        # list of possible values: ADM, Baidu, APNS, APNS_SANDBOX, GCM, MPNS, WNS
-        # each platform has a specific way to handle credentials
-        # this can also be used for dispatching message to the right platform
-        moto_response: CreatePlatformApplicationResponse = call_moto(context)
-        return moto_response
```

Comment:
I'd like to keep this operation or at the minimum the `TODO`, as it could a the next big work to do on SNS, as our emulation level there is very weak.

Reply:
Added back :+1: